recipes/larch-phylo/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+rm -rf build
+mkdir build
+cd build
+
+CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-"Release"}
+CMAKE_USE_USHER=${USE_USHER:-"ON"}
+CMAKE_NUM_THREADS=${CMAKE_NUM_THREADS:-"8"}
+MAKE_NUM_THREADS=${MAKE_NUM_THREADS:-"20"}
+LARCH_INCLUDE_TEST=${LARCH_INCLUDE_TEST:-"false"}
+LARCH_RUN_TEST=${LARCH_RUN_TEST:-"false"}
+
+echo "CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}"
+echo "CMAKE_USE_USHER: ${CMAKE_USE_USHER}"
+echo "CMAKE_NUM_THREADS: ${CMAKE_NUM_THREADS}"
+echo "MAKE_NUM_THREADS: ${MAKE_NUM_THREADS}"
+echo "LARCH_INCLUDE_TEST: ${LARCH_INCLUDE_TEST}"
+echo "LARCH_RUN_TEST: ${LARCH_RUN_TEST}"
```

Comment:
Please remove these indirections. It makes the recipe way more complicated.

Reply:
Sorry, we used theses internally on our github actions.  Removed indirection variables.